index.ts

```diff
@@ -205,6 +221,91 @@ export function createProxyCache<
     },
   };
 
+  const fetchers = {
+    fetchGuild: async (id: bigint) => {
+      // Return nothing if we don't want fetching
+      if (!options.fetchIfMissing?.guilds) return;
+      // Check if Guild is unavailable
+      if (unavailablesGuilds.has(id)) return;
+      // Fetch Guild through Helpers
+      const guild = await bot.helpers.getGuild(id);
+      // Return nothing if no Guild could be fetched
+      if (!guild) return;
```

Comment:
Is it possible for this to return nothing? I think either it will always return a Guild or it will throw an error so this code wont be needed imo